src/components/NoteDisplay/DeleteMode.js

```diff
@@ -0,0 +1,63 @@
+import React, { useContext } from "react";
+import Button from "../shared/Button";
+import styled from "styled-components";
+import { AppContext } from "../../contexts/AppContext";
+import globalTypes from "../../GlobalTypes";
+import { useHistory } from "react-router-dom";
+
+const CustomButton = styled(Button)`
+    position: relative;
+
+    &::before {
+        content: "";
+        position: absolute;
+        width: 100%;
+        height: 100%;
+        display: block;
+        background-color: rgba(0, 0, 0, 0.2);
+        top: 0;
+        left: 0;
+        opacity: 0;
+    }
+
+    &:hover::before {
+        opacity: 1;
+    }
+`;
+
+export default function DeleteMode({ noteId }) {
+    const { state, dispatch } = useContext(AppContext);
+    const history = useHistory();
+
+    return (
+        <>
+            <Button></Button>
+            <CustomButton
+                bg="crimson"
+                color="white"
+                onClick={() => {
+                    dispatch({
+                        type: globalTypes.deleteNote,
+                        id: noteId,
+                    });
+                    history.push("/trash");
+                }}
```

Comment:
Can you relocate this above the return statement. Kinda like:

```
const handleClick = () => {
     dispatch({
       type: globalTypes.deleteNote,
       id: noteId,
     });
     history.push("/trash");
};
```

For readability purposes? Kinda like like the return statement preferably should only contain the presentation as much as possible?